[Hackathon24] Translation: SQL workshop in Spanish

marianachavmed opened the pull request:
Copying the SQL folder for Spanish and doing half translation (activities 7-12, default and syntax_explanation folders).
Activities 1-6 will be completed by Alejandra Garay

ozhang22 commented:
@marianachavmed will Alejandra create a separate PR for activities 1-6 or will you be sharing this branch?

ozhang22 commented on content/espanol/sql_workshop/activity10/_index.md:
```diff
@@ -0,0 +1,85 @@
+{{% /notice%}}
+
+## Tarea 10: Ayuda a los habitantes a organizar la fiesta.
+The whole planet is invited! Help the citizens prepare for the party by finding out how many guests will be attending!
```
Translate

ozhang22 commented on content/espanol/sql_workshop/activity10/_index.md:
```diff
@@ -0,0 +1,85 @@
+<link rel="stylesheet" href="_activity10.css" type="text/css"></link>
+<script type="text/javascript" src="_activity10.js"></script>
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/UCxPAE0xw3I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```
For all the activity pages feel free to delete the youtube embed as it will not apply to the spanish workshop for now

marianachavmed replied:
Fixed

ozhang22 commented on content/espanol/sql_workshop/activity10/_index.md:
```diff
@@ -0,0 +1,85 @@
+      <h3 id = "prev"></h3>
+    </div>
+    <div style="clear: both;"></div>
+    <button class="button button1" onclick="sql()"> Enter </button>
```
Translate Enter (and similarly elsewhere)

marianachavmed replied:
I think "Enter" is valid in spanish too.

ozhang22 commented on content/espanol/sql_workshop/activity1/_activity1.js:
```diff
@@ -0,0 +1,30 @@
+        hide_boxes();
+    } else {
+    // If incorrect, give alert
+      alert("Try Again, Space Cadet!");
```
Translate all text within alert() function cals in JS files

marianachavmed replied:
Gotcha, will do a pass to translate the JS too, we were only focusing in the markup files.

ozhang22 commented on content/espanol/sql_workshop/activity12/_activity12.js:
```diff
@@ -0,0 +1,60 @@
+
+   if (trimmedPotential == "dr.phi")
+   {
+    document.getElementById("answer").innerHTML = "You found the inventor! Now he can combine the Legendary Totems of Fun into a more powerful Totem! This Totem unleashes a force field around the Planet of Fun! The planet is saved from the invaders!";
```
Translate text to the right of all instances of `.innerHTML = `

marianachavmed replied:
Fixed, please review again

ozhang22 commented on content/espanol/sql_workshop/activity9/_activity9.js:
```diff
@@ -0,0 +1,137 @@
+    update = current + 1;
+  }
+
+  document.getElementById(lock).innerHTML = update;
```
Except here, don't translate

marianachavmed replied:
Kept non quoted html changes

ozhang22 commented on content/espanol/sql_workshop/activity9/_index.md:
```diff
@@ -0,0 +1,117 @@
+    </div>
+
+    <div id="right_div">
+    <button class="button vault_button" id = "vault_button" onclick="confirm()"> Check </button>
```
Translate "Check"

marianachavmed replied:
Fixed

ozhang22 commented on content/espanol/sql_workshop/activity12/_activity12.js:
```diff
@@ -0,0 +1,60 @@
+
+   if (trimmedPotential == "dr.phi")
+   {
+    document.getElementById("answer").innerHTML = "!Encontraste al Inventor! ¡Ahora puedes unir los Tótems Legendarios del planeta Fun en un solo Tótem más poderoso! Este Tótem desata un campo de fuerza protectora alrededor del Planeta de la Diversión. ¡El planeta está a salvo de invasores!";
```
```suggestion
    document.getElementById("answer").innerHTML = "!Encontraste al Inventor! ¡Ahora puedes unir los Tótems Legendarios del planeta de la Diversión en un solo Tótem más poderoso! Este Tótem desata un campo de fuerza protectora alrededor del Planeta de la Diversión. ¡El planeta está a salvo de invasores!";
```

marianachavmed replied:
Changed to match the full spanish translation

ozhang22 commented on content/espanol/sql_workshop/activity12/_index.md:
```diff
@@ -0,0 +1,101 @@
+        <h3 id = "prev"></h3>
+      </div>
+    <div style="clear: both;"></div> 
+    <button class="button button1" onclick="sql()"> Enter </button>
```
Translate Enter

ozhang22 commented on content/espanol/sql_workshop/activity11/_activity11.js:
```diff
@@ -0,0 +1,178 @@
+  
+  /* Advance plot */
+  if(h == 1 && b == 1 && p == 1 && t == 1 && cake == 0 && ice == 0) {
+    alert("Congrats! You have food for the party!");
```
Translate

ozhang22 commented on content/espanol/sql_workshop/activity11/_activity11.js:
```diff
@@ -0,0 +1,178 @@
+function sql() 
+{
+  clearTable();
+  document.getElementById("sqlcommand").innerHTML = "ERROR INVALID INPUT";
```
Translate

ozhang22 commented on content/espanol/sql_workshop/activity10/_activity10.js:
```diff
@@ -0,0 +1,49 @@
+function sql() 
+{
+  clearTable();
+  document.getElementById("sqlcommand").innerHTML = "ERROR INVALID INPUT";
```
Translate

marianachavmed replied:
Fixed

ozhang22 commented on content/espanol/sql_workshop/basics_activity/_index.md:
```diff
@@ -0,0 +1,37 @@
+icon: ""
+draft: false
+---
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/l5BDCMyFHVI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```
Delete video

marianachavmed replied:
Removed the youtube video of all the activities

ozhang22 commented on content/espanol/sql_workshop/activity8/_index.md:
```diff
@@ -0,0 +1,84 @@
+        <h3 id = "prev"></h3>
+      </div>
+      <div style="clear: both;"></div> 
+      <button class="button button1" onclick="sql()"> Enter </button>
```
Translate Enter

ozhang22 commented on content/espanol/sql_workshop/activity8/_activity8.js:
```diff
@@ -0,0 +1,56 @@
+function sql() 
+{
+  clearTable();
+  document.getElementById("sqlcommand").innerHTML = "ERROR INVALID INPUT";
```
Translate

marianachavmed replied:
Fixed

ozhang22 commented on content/espanol/sql_workshop/activity7/_index.md:
```diff
@@ -0,0 +1,118 @@
+        <h3 id = "prev"></h3>
+      </div>
+      <div style="clear: both;"></div> 
+      <button class="button button1" onclick="sql()"> Enter </button>
```
Translate

marianachavmed replied:
We won't translate Enter in any of the instances as is commonly used in spanish too.

ozhang22 reviewed (approved):
Looks good!

marianachavmed commented:
@oliver-zhang-neuvo merged Alejandra's branch into mine, and addressed the comments you added in her [PR:https://github.com/NuevoFoundation/workshops/pull/488](https://github.com/NuevoFoundation/workshops/pull/494)

Can you see the changes, review and approve this PR instead? I can mark the other one as "draft"
